persistence-typed-tests/src/test/scala/org/apache/pekko/persistence/typed/scaladsl/EventSourcedStashOverflowSpec.scala

```diff
@@ -55,10 +55,11 @@ object EventSourcedStashOverflowSpec {
     SteppingInmemJournal.config("EventSourcedStashOverflow").withFallback(ConfigFactory.parseString(s"""
        pekko.persistence {
          typed {
-           stash-capacity = 1000 # enough to fail on stack size
+           stash-capacity = 2000 # enough to fail on stack size
```

Comment:
That's will cause more stack trace frame, that will event worst.

Maybe we could unstash with counter, after limitation, we will start another recursion(clear stack)

Reply:
I see, thank you.